src/components/TalentsListPage/TalentsList/TalentsList.jsx

```diff
@@ -1,5 +1,5 @@
 import styles from "../TalentsList/TalentsList.module.scss";
```

Comment:
you dont need to go out from your current directory before entering a sub directory, start your path from current directory instead of parent folder
```suggestion
import styles from "./TalentsList.module.scss";
```